examples/prometheus.yaml

```diff
@@ -57,6 +57,10 @@ kind: StatefulSet
 metadata:
   namespace: gmp-test
   name: prometheus-test
+  annotations:
+    # The emptyDir for the storage and config directories prevents cluster
+    # autoscaling unless this annotation is set.
```

Comment:
```suggestion
    # autoscaling unless this annotation is set. See
    # https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#what-types-of-pods-can-prevent-ca-from-removing-a-node
    # for details.
```

Reply:
Suggest adding comments for context. Also suggesting adding the relative context in https://github.com/GoogleCloudPlatform/prometheus-engine/blob/f753d7786f64c12be9b8c4408f2ab2c67705423f/cmd/operator/deploy/operator/10-collector.yaml#L31-L33 .

Let me know if you have any objections.